pal03377/Athena-New
# Show currently deployed PR in the playground

### Motivation and Context


It is difficult to determine whether a test deployment has already been successfully completed. Seeing the branch name helps with this problem, which is why we display it in test deployments of Artemis. 

### Description

This PR follows a similar approach and adds the current branch name to the footer of the playground.

### Steps for Testing

- Deploy this PR
- Go to https://athenetest1-03.ase.cit.tum.de/playground and observe the PR number, title, commit hash and last update time on the bottom left

### Screenshots

![screenshot-2023-07-17_001592](https://github.com/ls1intum/Athena/assets/9006596/a461437f-cd55-494c-afc9-977ed32c6bb6)
